migrator/migrations/m_206_to_m_207_bug_fix_default_policy_category/migration.go

```diff
@@ -0,0 +1,149 @@
+// Code generated by make bootstrap_migration generator. DO NOT EDIT.
+package m206tom207
+
+import (
+	"context"
+	"embed"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/migrator/migrations/m_203_to_m_204_openshift_policy_exclusions_for_4_5/schema"
+	"github.com/stackrox/rox/migrator/migrations/policymigrationhelper"
+	"gorm.io/gorm"
+)
+
+const (
+	startSeqNum = 203
```

Comment:
Take a look at the migrator README.  You should really get in the habit of using the bootstrap make target to start these migrations as it gets all the numbers set correctly.  As it is this number is incorrect, there is an import pointing to 203 and there are 2 or 3 additional files that aren't updated that the make target would have updated.

`DESCRIPTION="xxx" make bootstrap_migration`